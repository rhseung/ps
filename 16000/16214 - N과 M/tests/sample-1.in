4
1 1
3 4
28 43
370 205
