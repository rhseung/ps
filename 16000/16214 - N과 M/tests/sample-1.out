0
3
36
165
